examples/wave_tank/ex1/RigidBodyKinematics.cpp

```diff
@@ -58,6 +25,8 @@
 // IBTK INCLUDES
 
 // IBTK THIRD-PARTY INCLUDES
+#include "RigidBodyKinematics.h"
```

Comment:
is the class header a "third party" include?

Reply:
Oops :)

Reply:
Fixed.